draft-ietf-rtcweb-jsep.xml

```diff
@@ -3519,7 +3519,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             <t>If present, "a=msid" attributes MUST be parsed as
             specified in
             <xref target="I-D.ietf-mmusic-msid" />, Section 3.2, and
-            their values stored, ignoring any "appdata" field.</t>
+            their values stored, ignoring any "appdata" field. If no "a=msid"
+            attributes are present, a "default" msid-id value is created for
```

Comment:
By "default" do you literally mean the` id === "default"`, or figuratively some value chosen by the UA? Why would it need to be a specific string?

Wouldn't that limit UAs from using different ids for independent peer connections, either causing msid-less tracks from *all* peer connections on the page to end up in the same stream, or different streams with identical ids?

Reply:
When I used quotes around "default", the intention was the same as in https://tools.ietf.org/html/draft-ietf-mmusic-msid-16, Section 3.1; there's no official *default* MediaStream but we create a catch-all. However, I see that this could be interpreted the way you describe. Will reword.

Reply:
PTAL